### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by MarkPolivchuk**
conforming to configs for other open source projects

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bin
+build
+dist
```
Comment (iansu) [inline .eslintignore:3]: I don't think we have `dist` in this project.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "eslint-config-neo/config-backend",
+  "rules": {
+    "no-console": "off",
```
Comment (iansu) [inline .eslintrc:4]: Are these rule customizations needed?

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 100,
```
Comment (iansu) [inline .prettierrc:2]: This should be 120.

File: README.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 [![Build status](https://github.com/neofinancial/graphql-date-scalars/workflows/CI/badge.svg)](https://github.com/neofinancial/graphql-date-scalars/actions)
 ![TypeScript 3.5.3](https://img.shields.io/badge/TypeScript-3.5.3-brightgreen.svg)
 
-Scan files for credit card numbers
+Graphql Scalars for Date (YYYY-MM-DD), DateTime (YYYY-MM-DDTHH:MM:SSZ), and Time (HH:MM:SSZ)
```
Comment (iansu) [inline README.md:6]:
```suggestion
GraphQL Scalars for Date (YYYY-MM-DD), DateTime (YYYY-MM-DDTHH:MM:SSZ), and Time (HH:MM:SSZ)
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "graphql-date-scalars",
+  "description": "Scalars ",
+  "version": "1.1.1",
```
Comment (iansu) [inline package.json:4]: You should change this to `1.0.0` or `0.1.0`

File: package.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  "author": "Neo Financial Engineering <engineering@neofinancial.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/neofinancial/graphql-date-scalars"
+  },
+  "engines": {
+    "node": ">=10.0.0"
```
Comment (iansu) [inline package.json:12]: Let's use Node 12 as the minimum version.

File: package.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  },
+  "scripts": {
+    "build": "NODE_ENV=production ncc build src/app.ts -o build --minify --external update-notifier",
```
Comment (iansu) [inline package.json:15]: We don't want to use ncc in this project.

File: package.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    "eslint": "6.8.0",
+    "eslint-config-neo": "0.5.2",
+    "husky": "4.2.1",
+    "jest": "^25.1.0",
+    "lint-staged": "10.0.2",
+    "prettier": "1.19.1",
+    "rimraf": "3.0.0",
+    "ts-jest": "^25.0.0",
+    "ts-node": "8.6.2",
+    "typescript": "3.7.5"
```
Comment (iansu) [inline package.json:54]: Make sure to bump all the dependencies.

**Review by iansu — CHANGES_REQUESTED** [review]

File: package.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  "devDependencies": {
+    "@types/jest": "26.0.14",
+    "@types/node": "14.11.10",
+    "chalk": "4.1.0",
+    "eslint": "7.11.0",
+    "eslint-config-neo": "0.5.2",
+    "husky": "4.3.0",
+    "jest": "^26.5.3",
+    "lint-staged": "10.4.2",
+    "prettier": "2.1.2",
```
Comment (iansu) [inline package.json:50]: Our ESLint config doesn't currently work with Prettier 2 or TypeScript 4.

**Review by iansu — CHANGES_REQUESTED** [review]

File: LICENSE [hunk 10]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Neo Financial Technologies Inc., Dirk-Jan Rutten
```
Comment (iansu) [inline LICENSE:3]:
```suggestion
Copyright (c) 2020 Neo Financial Technologies Inc.
Copyright (c) 2017 Dirk-Jan Rutten
```

**Review by iansu — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 10 above)
Comment (iansu) [reply LICENSE:3]: I don't actually know the correct way to do this but some Googling resulted in this.

**Comment by krisread** [comment]
@iansu  @MarkPolivchuk   Hey why is this blocked, is this worth keeping open, its 20+ days old, is there a major issue...?  It looks like the last suggestion was not worth blocking over?

**Comment by iansu** [comment]
I do think getting the license correct is important. That's the only remaining change.

**Review by iansu — APPROVED** [review]

**Pull request merged**